imod/tests/test_mf6/test_mf6_wel.py

```diff
@@ -137,35 +138,51 @@ def test_to_mf6_pkg__high_lvl_transient(basic_dis, well_high_lvl_test_data_trans
     np.testing.assert_equal(mf6_ds["rate"].values, rate_expected)
 
 
-def test_clip_box__high_lvl_stationary(well_high_lvl_test_data_stationary):
+@pytest.mark.parametrize(
```

Comment:
it would be nice to add test cases where for example layer_min is not None, but layer_max is. And vice versa. Or is that combination not allowed?
Also the parameters time_min and time_max could be included in the testing

Reply:
I see we are now also using testdata where bottom, top are not grids. In the fixtures: 

```python
@pytest.fixture
def parameterizable_basic_dis(request):
...
    top = xr.DataArray(z[:-1], coords={"layer": layers})
    bottom = xr.DataArray(z[1:], coords={"layer": layers})
...
```

This is possible, but doesn't cover most of the use cases. Most of the applications have ``top`` and ``bottom`` specified as grids. We should cover that in our tests as well.